test/uk/gov/hmrc/serviceconfigs/WireMockEndPoints.scala

```diff
@@ -20,7 +20,7 @@ import java.net.ServerSocket
 
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock._
-import com.github.tomakehurst.wiremock.client.{MappingBuilder, ResponseDefinitionBuilder, WireMock}
+import com.github.tomakehurst.wiremock.client.{ResponseDefinitionBuilder, WireMock}
```

Comment:
Actually, this code doesn't appear to be used at all in the project. Shall I just delete `WireMockEndPoints.scala`?

Reply:
Yes, just remove it. It should have been replaced by helpers from `http-verbs-test`